### ISSUE TITLE
Change the URL of the Deployment API Docs Modules to Reflect BallerinaX

### ISSUE THREAD

**Issue by praneesha** [ballerina-platform/ballerina-dev-website#1097]
**Description:**
We need to change the URL of the Deployment API Docs Modules to indicate that they are from the "BallerinX" GitHub organisation as shown below.

<img width="1680" alt="Screenshot 2020-07-23 at 11 16 35" src="https://user-images.githubusercontent.com/11707273/88256348-b92da500-ccd8-11ea-91e3-202716eb837f.png">

This should be done for both 1.2.x and Swan Lake versions.

**Suggested Labels:**

**Suggested Assignees:**

**Affected Product Version:**

**OS, Browser, other environment details and versions:**    

**Steps to reproduce:**

**Related Issues:**

**Comment by hemikak** [comment]
In that case, `awslambda` and `azure.functions` should not be in https://ballerina.io/learn/api-docs/ballerina/ page as below as they are from `ballerinax`:
<img width="954" alt="Screen Shot 2020-07-23 at 11 39 54 AM" src="https://user-images.githubusercontent.com/4398820/88256711-994ab100-ccd9-11ea-9e39-2f0796a22d59.png">

They should be in a page called https://ballerina.io/learn/api-docs/ballerinax/ . If so how can users access https://ballerina.io/learn/api-docs/ballerinax/ link ?

**Comment by praneesha** [comment]
@lafernando - WDYT?

**Comment by lafernando** [comment]
Yes, they should not be there under the path "ballerina". We need another high level menu to select between organizations: "ballerina" and "ballerinax".

**Comment by praneesha** [comment]
@hemikak @Thimuth -  Can we make the 2 URLs be like this?

- https://ballerina.io/learn/ballerina/api-docs/
- https://ballerina.io/learn/ballerina/api-docs/

**Comment by hemikak** [comment]
Need to check the doc command and also how to pack it to the distribution.
